### ISSUE TITLE
Suggested improvements to Week 3 lab

### ISSUE THREAD

**Issue by arthur-e** [OscarFHC/EAS538_2018#17]
- [ ] `legend()` function in the code examples encourages students to use this in their work, but the legend blocks out the histogram on smaller screens; remove `legend()` entirely?
- [ ] Students are encourage to remember how to read in data from the Week 2 lab, except they used `read.table()` last week and are calling it with the same arguments set, which results in a really weird data frame (they should `read.csv()` instead, for this week's data);

**Comment by mehajain13** [comment]
1) Sure removing 'legend' sounds good to me
2) ah right, let's provide them with one line of code that is apdata=read.csv('data.csv') and we can say this is another way to read data when your file is in .csv format

Thanks! I'll assume you'll make these changes but let me know if you want me to.
m

**Comment by arthur-e** [comment]
I realized that without the `legend()` there's really no reason to calculate the `avg` and `SD` so, as a middle ground, I've added a formatting parameter that I *think* will remove the box and background of the legend, making overplotting less of a concern.

I say "I think" because it's apparent this is platform-dependent. When I make my plotting window really small, the legend doesn't cover the histogram, as I observed it to do on students' computers yesterday.

Only so much we can do! Thanks for considering these changes.
